src/main/controlMod.F90

```diff
@@ -257,6 +257,11 @@ subroutine control_init(dtime)
 
     namelist /clm_inparm/ use_biomass_heat_storage
 
+    namelist /clm_inparm/ use_hillslope
+
+    namelist /clm_inparm/ downscale_hillslope_meteorology
```

Comment:
It seems like downscale_hillslope_meteorology and use_hillslope_routing could be in the Hillslope specific namelist rather than here.

use_hillslope is an overall control variable, so needs to be here, but it looks like to me these two are only used if use_hillslope is on.

Reply:
By having namelist controls in the most specific location as possible, it makes it easier to change parameterization packages (add new versions, or remove old ones). The control variables at this level are kind of out of control (too many of them), so I like to make them specific if it can be done.